### DISCUSSION TITLE
Write in "App logs"

### DISCUSSION THREAD

**Discussion by johndodev** [caprover/caprover#1725] [Q&A]
Hello,

I wonder how to log something in the "App logs" in the "deployment" tab.

More details, I use caprover for a webserver of multiples projects. App logs is showing apache acces logs. I also manage to use systemd in project app to create a service to consume async job (rabbitmq like). I'd like to output systemd output to that "App logs", but I don't know how. I don't even succeeded to log it in a file, but anyways, is it possible to do something like that ? 

`echo "Hello" > [somewhere in app logs]`

**Comment by githubsaturn** [comment]
App logs come from `docker service logs` so whatever is printed in the docker logs shows up in docker logs. 

For apache logs, it's specifically using the apache logs:
https://hub.docker.com/layers/library/php/apache/images/sha256-0f6a5fedb609604fbc46af704525fbdff045de469d1bb6baf26befc3011581ad?context=explore

It's really dependent on the image. If you want custom logs, you need to build your own custom docker image.
